app/controllers/groups/made_available_controller.rb

```diff
@@ -3,35 +3,15 @@ class MadeAvailableController < ApplicationController
     before_action :build_request, :data_check
 
     ROUTE_MAP = {
-      index: proc { |params| ParliamentHelper.parliament_request.group_layings_index.set_url_params({ group_id: params[:group_id] }) }
+      index: proc { |params| ParliamentHelper.parliament_request.group_laid_papers_index.set_url_params({ group_id: params[:group_id] }) }
     }.freeze
 
     def index
-      @group, @layings = FilterHelper.filter(@api_request, 'Group', 'Laying')
-      @layings = @layings.sort_by(:date, :graph_id).reverse
+      @group, @laid_papers = FilterHelper.filter(@api_request, 'Group', 'LaidThing')
+      @laid_papers = @laid_papers.sort_by(:date, :graph_id).reverse
```

Comment:
You don't need this line for sorting here as you sort later on with the factory.